src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesStrictSpec.scala

```diff
@@ -34,72 +34,83 @@ class PostRoutesStrictSpec extends FunSpec with BeforeAndAfter with Matchers {
           response.status should be(StatusCodes.OK)
           val entity = entityString(response)
           val route = entity.parseJson.convertTo[RouteOut]
-          route.id should be (1)
+          route.id should be(1)
           route.name should be(RouteName(routeName))
           route.ownedByTeam should be(TeamName("team1"))
           route.createdBy should be(UserName("user~1"))
         }
 
-        describe("when a token with the write_regex scope is provided") {
-          val token = WRITE_REGEX_TOKEN
-
-          it("should create the new route") {
-            val routeName = "strict_route_2"
-            val response = postSlashRoutesStrict(routeName, token)
-            response.status should be(StatusCodes.OK)
-            val entity = entityString(response)
-            val route = entity.parseJson.convertTo[RouteOut]
-            route.id should be (1)
-            route.name should be(RouteName(routeName))
-          }
+        it("should not create more routes") {
+          val routeName = "route_regex_1"
+          val response = postSlashRoutesStrict(routeName, token)
+
+          val routesResponse = getSlashRoutes(READ_TOKEN)
+          response.status.shouldBe(StatusCodes.OK)
+          val entity = entityString(routesResponse)
+          val routes = entity.parseJson.convertTo[Seq[RouteOut]]
+          routes.size should be(1)
         }
       }
 
-      describe("failure") {
+      describe("when a token with the write_regex scope is provided") {
+        val token = WRITE_REGEX_TOKEN
+
+        it("should create the new route") {
+          val routeName = "strict_route_2"
+          val response = postSlashRoutesStrict(routeName, token)
+          response.status should be(StatusCodes.OK)
+          val entity = entityString(response)
+          val route = entity.parseJson.convertTo[RouteOut]
+          route.id should be(1)
+          route.name should be(RouteName(routeName))
+        }
+      }
+    }
 
-        describe("when an invalid name is provided") {
-          val token = WRITE_STRICT_TOKEN
+    describe("failure") {
 
-          it("should create the new route") {
-            val routeName = "invalid-strict-route-name"
-            val response = postSlashRoutesStrict(routeName, token)
-            response.status should be(StatusCodes.BadRequest)
-          }
+      describe("when an invalid name is provided") {
+        val token = WRITE_STRICT_TOKEN
+
+        it("should create the new route") {
```

Comment:
Description does not match the outcome of the test. BadRequest is returned. I expect that route is not created in this case.